### PR TITLE
Make `micro` the default dataset size 

### PR DESCRIPTION
**Pull request by mivasconcelos**
We want to use the `micro` size since the `100k-2p` is too heavy for an s00 cluster. We should also reduce the size defined for the s00 for consistency.

Test plan:
- connect to the workspace without the `martech` dataset loaded
- create the DB
- go to the pipelines page on Customer Portal and ensure they were created from the `1k-2p` bucket
screenshot-2023-03-21-15-09-09

### CODE REVIEW

File: web/src/scalefactors.ts [hunk 1]
```diff
@@ -21,7 +21,7 @@ export const ScaleFactors: Array<ScaleFactor> = [
   {
     name: "s00",
     maxRows: 10_000_000,
-    prefix: "v2/100k-2p",
+    prefix: "v2/10k-2p",
```
Comment (sandeshvijayraj) [inline web/src/scalefactors.ts:24]: I think micro is 1k-2p ?

Comment (sandeshvijayraj) [inline web/src/scalefactors.ts:24]: even tiny is 10k-2p now !! Do we want to modify tiny ???

**Review by TitoGrine — APPROVED** [review]

File: web/src/scalefactors.ts (hunk 1 above)
Comment (sandeshvijayraj) [reply web/src/scalefactors.ts:24]: what about s00? what I was asking is

s00 and tiny now have the same prefix is that okay? Not sure though how this part works

Comment (mivasconcelos) [reply web/src/scalefactors.ts:24]: oh, got it. It's ok, we're not currently using it, it's just an informative guideline we mention in the readme file. s00 is the smaller cluster size we have, others are just an addition.

**Pull request merged**